tests/wpt/web-platform-tests/dom/nodes/Document-createEvent.html

```diff
@@ -151,4 +151,8 @@
     }, 'Should throw NOT_SUPPORTED_ERR for pluralized non-legacy event interface "' + eventInterface + 's"');
   }
 });
+
+test(function() {
+  assert_equals(document.createEvent('storageevent'), "");
+}, "document.createEvent('storageevent') does not propagate the document's URL.")
```

Comment:
What you are asking me is way more involved than a simple Servo fix though. Should I file that we don't test that on the WPT repos?